homeassistant/components/azure_data_explorer/strings.json

```diff
@@ -0,0 +1,26 @@
+{
+  "config": {
+    "step": {
+      "user": {
+        "title": "Setup your Azure Data Explorer integration",
+        "description": "Enter connection details.",
+        "data": {
+          "clusteringesturi": "Cluster Ingest URI",
+          "database": "Database name",
```

Comment:
Are there no existing strings for database and table?

Reply:
No, I could not find any